src/components/Select/Select.tsx

```diff
@@ -127,38 +127,41 @@ export const Select: React.FC<SelectProps> = ({
   ]);
 
   return (
-    <div
-      id={`Select--${idRef}`}
-      className={classNames('Select', { 'Select--disabled': disabled }, className)}
-      ref={selectRef}
-      {...rest}
-    >
-      <Clickable
-        id={`Select__value--${idRef}`}
-        className='Select__value'
-        disabled={disabled}
-        onClick={handleToggle}
-        aria-haspopup='listbox'
-        aria-expanded={state.mode === Mode.Open}
-        aria-labelledby={`Select__label--${idRef} Select__value--${idRef}`}
-        type='button'
+    <>
+      <input id={idRef} type='hidden' value={value} />
```

Comment:
Might be better to put this inside the `div`, I'm not sure if this would break any styles otherwise, e.g. if you have a `<Select>` in a `<Stack>`.
Probably not, but I still feel like it's better to put it in the `div`.